parser_library/CMakeLists.txt

```diff
@@ -250,6 +250,11 @@ if(BUILD_FUZZER)
 
     set_target_properties(library_fuzzer PROPERTIES LINK_FLAGS "-fsanitize=fuzzer")
 	set_target_properties(library_fuzzer PROPERTIES COMPILE_FLAGS "-DANTLR4CPP_STATIC")
+    
```

Comment:
Is the file copied properly every time it is changed?

Reply:
If the input file is modified the build system will re-run CMake to re-configure the file and generate the build system again.